runtime/lua/vim/_system.lua

```diff
@@ -230,6 +230,8 @@ local function default_handler(stream, text, bucket)
   end
 end
 
+local is_win = vim.fn.has('win32') == 1
```

Comment:
Is there also win64?

Reply:
No. I think this just means all windows.

Reply:
There is a win64 but win32 includes 32 and 64 bit, so everything is fine